ext/mysql2/extconf.rb

```diff
@@ -66,6 +66,12 @@ def add_ssl_defines(header)
   $LDFLAGS << " -L#{openssl_location}/lib" if openssl_location
 end
 
+if RUBY_PLATFORM =~ /darwin/ && system('command -v brew')
+  zstd_location = `brew --prefix zstd`.strip
+
+  $LIBPATH << "#{zstd_location}/lib" unless zstd_location.empty?
```

Comment:
If you found `.empty?` necessary vs. falsiness of the empty string, then the `if openssl_location` clause above should change to match -- did you test to see if brew's output when the package is not installed is in fact "truthy"?

```suggestion
  $LIBPATH << "-L#{zstd_location}/lib" unless zstd_location.empty?
```
Also I notice the openssl pattern above adds `-L` -- I think these ought to match? But if I'm missing something let me know!

Reply:
> vs. falsiness of the empty string

Empty strings aren't falsy in Ruby.

> Also I notice the openssl pattern above adds -L -- I think these ought to match? 

They're not adding to the same variables. `$LIBPATH` takes paths, `$LDFLAGS` takes flags. I guess I can rewrite the OpenSSL one to use LIBPATH and properly do its error checks.

Reply:
Done.

Reply:
🤦 quite right. So what's that line above testing? Seems like a bogon, unless there's some long-lost mystery behavior of yore (old code, after all)

Reply:
It wasn't testing anything. Was likely just an oversight.